client/src/components/Pages/Experience/constants.js

```diff
@@ -0,0 +1,104 @@
+import {
+  experienceIcicle,
+  experienceMBAS,
+  experienceEnixan,
+  experienceBigfinite,
+  experienceCornerjob,
+} from '../../../images';
+
+export const experiences = [
```

Comment:
Maybe it is better to keep this kind of info in the database instead of hardcoded data. But I think for the moment it is ok to have it like this.